package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@webex/webrtc-core",
-  "version": "2.4.0",
+  "version": "2.4.1",
```

Comment:
No need to change this, it will be changed automatically when we generate a release :)